### PR TITLE
[IMP] point_of_sale: cashbox opening popup.

### PR DESCRIPTION
**Pull request by rhe-odoo**
Move the cashbox opening control from the left panel of the product screen widget to a popup that is raised at the opening of the session if its state is: opening_control.

task-id: 2321973
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by odony** [comment]
When merging in stable you should not delete JS files that are used by asset bundles, as the bundles will be recompiled upon source code update, yet with the old definition of the bundles. Usually we keep empty files with just a comment, and hard-delete them in master.

**Comment by pimodoo** [comment]
@odony Thanks, I just rebased it and was about to review, it is  because two files were renamed, so we'll rename them only in master.

File: addons/point_of_sale/static/src/xml/Screens/ProductScreen/ProductScreen.xml [hunk 1]
```diff
@@ -5,29 +5,24 @@
...
-                                <NumpadWidget activeMode="state.numpadMode" />
-                            </div>
+                    <OrderWidget/>
+                        <div class="pads">
+                        <div class="control-buttons">
+                            <t t-foreach="controlButtons" t-as="cb" t-key="cb.name">
+                                <t t-component="cb.component" t-key="cb.name" />
+                            </t>
+                        </div>
+                            <div class="subpads">
```
Comment (pimodoo) [inline addons/point_of_sale/static/src/xml/Screens/ProductScreen/ProductScreen.xml:15]: check the indentattion, correctly align lines below

**Comment by pimodoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**